scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -1905,7 +1911,7 @@ void BPF_STRUCT_OPS(lavd_enqueue, struct task_struct *p, u64 enq_flags)
 
 static bool is_kernel_task(struct task_struct *p)
 {
-	return p->flags & PF_KTHREAD;
+	return !!(p->flags & PF_KTHREAD);
```

Comment:
Doesn't really matter but `!!` doesn't make any difference here. The implicit bool casting for the return value does the same thing.